integration-tests/features/steps/gremlin.py

```diff
@@ -28,18 +29,25 @@ def gremlin_search_vertexes_for_the_ecosystem(context, ecosystem):
     post_query(context, query)
 
 
-@when('I ask Gremlin to find the package {package:S} in the ecosystem {ecosystem}')
+@when('I ask Gremlin to find all versions of the package {package:S} in the ecosystem {ecosystem}')
 def gremlin_find_package(context, package, ecosystem):
-    """Try to find the package in the selected ecosystem."""
-    query = Query().has("pecosystem", ecosystem).has("ppackage", package)
+    """Try to find all versions of the given package in the selected ecosystem."""
+    query = Query().has("pecosystem", ecosystem).has("pname", package)
     post_query(context, query)
 
 
 @when('I ask Gremlin to find the package {package:S} version {version} in the ecosystem '
       '{ecosystem}')
 def gremlin_find_package_version(context, package, version, ecosystem):
     """Try to find the package with version in the selected ecosystem."""
-    query = Query().has("pecosystem", ecosystem).has("ppackage", package).has("version", version)
+    query = Query().has("pecosystem", ecosystem).has("pname", package).has("version", version)
+    post_query(context, query)
+
+
+@when('I ask Gremlin to find the package {package:S} in the ecosystem {ecosystem}')
+def gremlin_find_package(context, package, ecosystem):
+    """Try to find the package in the selected ecosystem."""
+    query = Query().has("ecosystem", ecosystem).has("name", package).has("vertex_label", "Package")
```

Comment:
Again, no need to provide vertex_label here. Just 
`query = Query().has("ecosystem", ecosystem).has("name", package)` should suffice

Reply:
good to know, thanks @miteshvp

Reply:
fixed this one, thanks